web/src/click_handlers.js

```diff
@@ -573,6 +573,24 @@ export function initialize() {
             get_target_node,
             check_reference_removed,
         );
+
+        $(".status-emoji-name").off("mouseenter").off("mouseleave");
```

Comment:
I notice that the commit message is missing a blank line between the title and body. Every git commit should have such a blank line.

Reply:
Should be good to go now.